src/components/_modal/pages/deleteOrder/DeleteOrder.page.js

```diff
@@ -0,0 +1,86 @@
+import Top from "../base/top";
+import styles from "./DeleteOrder.module.css";
+import Translate from "@/components/base/translate";
+import Text from "@/components/base/text";
+import Button from "@/components/base/button";
+import { handleCancelOrder } from "./utils";
+
+function DeleteOrder({ context, modal }) {
+  const {
+    label,
+    mobile,
+    isReadyToPickup,
+    orderId,
+    agencyId,
+    orderMutation,
+    onClose,
+  } = context;
+
+  function closeModal() {
+    mobile ? modal.prev() : modal.clear();
+  }
+
+  async function onCancelOrder() {
+    await handleCancelOrder(orderId, agencyId, orderMutation);
+    if (!orderMutation.error) {
+      onClose({ success: true });
+    } else {
+      onClose({ success: false });
+    }
+    modal.clear();
+  }
+
+  return (
+    <article className={styles.deleteOrder}>
+      <Top title={label} titleTag="h4" dataCy="modal-header" />
+      <hr />
+      <div className={styles.container}>
+        <Text type="text2" tag="div">
+          {Translate({
+            context: "profile",
+            label: "delete-order-confirmation",
+          })}
+        </Text>
+        <Text type="text2" tag="div" className={styles.text}>
```

Comment:
Er der en grund til at bruge div i stedet for p-tag her? Ved ikke om det vil give nogle sideeffekter i forhold til tilgængelighed/SEO.

Reply:
godt set, det er rettet